### DISCUSSION TITLE
dynamic mac address

### DISCUSSION THREAD

**Discussion by webysther** [seriousm4x/UpSnap#77] [General]
What happens when a device have a dynamic mac address?

**Comment by seriousm4x** [comment]
Pinging, port scan and shutdown will still work because the IP is used. However, if the MAC changes, waking up will no longer work. Normally, a MAC does not change unless you use a VM and change the MAC. Some devices (e.g. iOS) support a "private MAC address" which can change. If this is the case, you have to change the MAC in the web interface.
